src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -5,31 +5,31 @@
     :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-btn-dropdown` : 'ec-btn-dropdown'"
   >
     <ec-btn
-      :is-disabled="isDisabled"
+      :is-disabled="props.isDisabled"
```

Comment:
Props are automatically available in the template, so you don't have to update their references in the template. You can remove all occurrences of `props.*` from the template.

Reply:
corrected